### PR TITLE
Removal of 'we'

### PR DESCRIPTION
**Pull request by neilnutt**
-It is my preference that OH has no "we" so I have removed.
-Removal of extra line breaks
-"created by one of the OH contributors" gave the impression (possibly somewhat accurately) that it is all reliant on only one person.

### CODE REVIEW

**Comment by faph** [comment]
The code is from 1 contributor only. This applies to the flood estimation library and OH Auto Statistical.

I would keep the paragraphs seperately for readibility.

**Comment by mikerspencer** [comment]
The line breaks create the paragraph spacing. Not entirely sure separate
paragraphs were needed, but I'm happy either way.

I agree with OH contributors, but it's an author statement that we want
participation of wider community. In this case we should remain for clarity
of understanding (who is making the statement) and readability. Agree with
other tweaks (nearly). Will very minorly adjust.

I think it's nearly time to draw a line under the doc and call it ready.

**Comment by mikerspencer** [comment]
Hmm I prefer the original of the wider hydro community. Should have checked pre commit!

**Comment by neilnutt** [comment]
Florenz - I'm not arguing with you that it is your work but it is not the
image I think we should be portraying.  If you go back to the very start
there will be some of my feh functions in the floodestimation library in
there but so little it is trivial and it would have been easier for you to
do by yourself.  Ultimately the floodestimation library is built on lots of
other open source work by others, i.e. lmonents etc.

Agree that it is time to draw to an end.  Just need to decide where the
line breaks go to define the paragraphs, previously we had 1 & 2 sentence
paragraphs which were hard to follow.

**Comment by faph** [comment]
I have made some suggestions for tweaks. Please consider these as suggestions only. I have kept them on separate commits with explanation so please cherry-pick as required.
